### DISCUSSION TITLE
Сутності Person, User, Party, Employee

### DISCUSSION THREAD

**Discussion by Vitaliy-1** [Vitaliy-1/openHealth#50] [Ideas]
Різниця між основними сутностями, які відображають користувачів

**Comment by biben1989** [comment]
User є основною сутністю, яка служить відправною точкою для інших дочірніх сутностей, таких як Person та Party. Це точка входу в систему, що використовується для авторизації та ідентифікації користувача.

**Опис сутностей:**

**User**: Головна сутність, яка представляє користувача системи. Вона включає інформацію, необхідну для входу в систему та доступу до її функціоналу. Сутність User є центральним елементом, від якого залежить вся інша інформація, пов'язана з користувачем.

**Person**: Ця сутність містить загальну інформацію про користувача, таку як ім'я, дата народження, контактні дані тощо. Вона надає базові відомості, які ідентифікують користувача як фізичну особу.

**Party**: Ця сутність зосереджена на документах, освіті та професійному досвіді користувача. Вона включає в себе інформацію про отримані дипломи, сертифікати, трудову діяльність і будь-які інші офіційні документи, що можуть бути важливі для користувача в системі.

**Employee**: Сутність, яка описує роль користувача в межах організації, його посаду, а також до якого підрозділу або закладу відноситься ця посада. Це дозволяє зрозуміти, в якому контексті користувач працює, яка його відповідальність і де він застосовує свої навички та знання.
